src/rc-socket.js

```diff
@@ -68,6 +68,11 @@ var RcSocket = function (url, protocols) {
     this.unload = true;
   }.bind(this);
 
+  // Protect against network/ip changes.
+  window.addEventListener('offline', function () {
+    this.refresh();
```

Comment:
Perhaps could use some better commenting. It will refresh immediately.

Refresh calls `close `directly on the WebSocket (`this.ws`). This will in turn hit the [ws.onclose](https://github.com/firstopinion/rc-socket.js/blob/master/src/rc-socket.js#L118) handler which calls [_reconnect](https://github.com/firstopinion/rc-socket.js/blob/master/src/rc-socket.js#L126).